build/common/installer/scripts/tomlparser.rb

```diff
@@ -24,6 +24,7 @@
 @containerLogsRoute = "v2" # default for linux
 @adxDatabaseName = "containerinsights" # default for all configurations
 @logEnableMultiline = "false"
+@stacktraceLanguages = "go,java,python"
```

Comment:
why the dotnet not here? is it dotnet will not be default? if yes, this might impact existing customers who are using dotnet, right?

Reply:
will add